### DISCUSSION TITLE
`v3.0.0` releases

### DISCUSSION THREAD

**Discussion by Malix-Labs** [lisonge/vite-plugin-monkey#58] [Q&A] [answered]
When do you think [`v3.0.0`](https://github.com/lisonge/vite-plugin-monkey/releases/tag/v3.0.0) releases?

**Comment by lisonge** [comment, marked as answer]
sourcemap is temporarily unavailable, I am still learning how to merge it correctly

**Reply by lisonge** [reply]
sourcemap will be released in the future version
